spec/query_spec.rb

```diff
@@ -17,12 +12,14 @@
     end
   end
 
+  let(:client_double) { double("Client") }
+
   before(:each) do
-    @client = double('client')
-    allow(@client).to receive(:api_key).and_return('api_key')
+    allow(client_double).to receive(:api_key).and_return("api_key")
+    allow(client_double).to receive(:endpoint_url).and_return("https://example.com")
```

Comment:
Couldn't the endpoint_url be "https://api.twingly.com/..." instead? This caught my attention since I had to think twice what `endpoint_url` actually is used for.